src/sql/workbench/browser/editor/profiler/profilerInput.ts

```diff
@@ -126,14 +130,40 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 	}
 
 	public override getName(): string {
-		let name: string = nls.localize('profilerInput.profiler', "Profiler");
+		let name: string = ProfilerInput.PROFILERNAME;
 		if (!this.connection) {
 			return name;
 		}
-		name += ': ' + this.connection.serverName.substring(0, 20);
+		if (this.connection.connectionName) {
+			name += ': ' + this.connection.connectionName.substring(0, 20);
+		}
+		else {
+			name += ': ' + this.connection.serverName.substring(0, 20);
+		}
 		return name;
 	}
 
+	public override getTitle(verbosity?: Verbosity): string {
+		let fullTitle = ProfilerInput.PROFILERNAME;
+		if (this.connection) {
+			let baseName = this.connection.serverName + ':' + this.connection.databaseName;
+			let advancedOptions = this._connectionService.getNonDefaultOptions(this.connection);
+			fullTitle = fullTitle + ': ' + baseName + advancedOptions;
```

Comment:
Isn't adding advanced options a little too much information for the tooltip? I am assuming it would be a lot to hear if voice over is turned on.

Reply:
It is a lot, but it would clutter the title if we appended the advanced options to it. 

If there is any specific information related to connections that can only be gained from a connection profile for the profiler extension, then yes, the advanced options are needed to distinguish between different profiles. 

However, if they display the same information no matter the advanced options, then this addition is not needed. Please clarify if this is the case.

Reply:
As discussed, probably adding session name instead of the advanced options would make more sense for the Profiler.